docroot/modules/custom/mass_content/mass_content.services.yml

```diff
@@ -13,3 +13,6 @@ services:
     tags:
       - { name: event_subscriber }
     weight: 200
+  mass_content.image_style_warmer:
```

Comment:
Have you seen https://www.drupal.org/project/warmer ? This could be nice as a plugin contributed to that project.

Reply:
Agree, will take a look, thanks